config/locales/lib/en.yml

```diff
@@ -108,7 +108,9 @@ en:
           favicon:
             description: Used as the site favicon
           ga_tracking:
-            description: Google Analytics Tracking ID, e.g. UA-71991000-1
+            description: Google Universal Analytics Tracking ID, e.g. UA-XXXXXXXX-1
```

Comment:
Whoa: Dev's GA tracking ID was in the docs and showing up on all Forem's admin pages! 😱